plugins/svg/src/LinearFeatureDisplay/model.ts

```diff
@@ -0,0 +1,85 @@
+import { ConfigurationReference, getConf } from '@jbrowse/core/configuration'
+import { getParentRenderProps } from '@jbrowse/core/util/tracks'
+import { MenuItem } from '@jbrowse/core/ui'
+import { BaseLinearDisplay } from '@jbrowse/plugin-linear-genome-view'
+import VisibilityIcon from '@material-ui/icons/Visibility'
+
+import { types, Instance } from 'mobx-state-tree'
+import { AnyConfigurationSchemaType } from '@jbrowse/core/configuration/configurationSchema'
+
+const stateModelFactory = (configSchema: AnyConfigurationSchemaType) =>
+  types
+    .compose(
+      'LinearFeatureDisplay',
+      BaseLinearDisplay,
+      types.model({
+        type: types.literal('LinearFeatureDisplay'),
+        showLabels: types.optional(types.boolean, true),
+        displayMode: types.optional(types.string, 'normal'),
```

Comment:
rename this to something other than displayMode probably, maybe compactness

Reply:
maybe "density"?

Reply:
might leave as is, unless needed to change